### DISCUSSION TITLE
[2022/09/13] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#222] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/221 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by kobayang** [comment]
## [tRPC](https://trpc.io/)

- TypeScript の RPC Server を記述するフレームワーク。
- Code First なアプローチで、Input/Resove で実装されたレスポンスが型として生成される。
- GraphQL に比べて、スキーマを経由せずに、サーバー側で生成された型をフロントエンド側で直接使うことができる。
